akka-persistence-query/src/test/scala/akka/persistence/query/journal/leveldb/EventsByTagSpec.scala

```diff
@@ -17,9 +17,12 @@ import akka.testkit.ImplicitSender
 import akka.persistence.query.NoOffset
 
 object EventsByTagSpec {
-  val config = """
+  val config = s"""
     akka.loglevel = INFO
+    akka.stream.materializer.max-input-buffer-size = 1
+    akka.stream.materializer.initial-input-buffer-size = 1
```

Comment:
Why set this for all test cases, if it is needed for a specific test, set it with attributes instead?

Reply:
Ah the test changed to not rely on this, removing